.github/workflows/website.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Set up R
         if: steps.check-rmd.outputs.count != 0
-        uses: r-lib/actions/setup-r@master
+        uses: r-lib/actions/setup-r@v2-branch
```

Comment:
was this breaking anything? I agree it's good to keep in sync with main carpentries regardless
@ErinBecker can you confirm this change?

Reply:
This didn't break any of the notebooks/class code, but it did break the CI checks below which was why I updated it. It's totally fine to move this change to a different branch, etc, if there is a better workflow or need to separate it from the other changes.